zanata-adapter-properties/src/main/java/org/zanata/adapter/properties/PropWriter.java

```diff
@@ -68,32 +132,18 @@ private static void write(final Resource doc, final File baseDir,
         storeProps(props, baseFile, utf8);
     }
 
-    public static void write(Resource srcDoc, final TranslationsResource doc,
-            final File baseDir, String bundleName, String locale,
-            boolean createSkeletons) throws IOException {
-        write(srcDoc, doc, baseDir, bundleName, locale, false, createSkeletons);
-    }
-
-    public static void writeUTF8(Resource srcDoc,
-            final TranslationsResource doc, final File baseDir,
-            String bundleName, String locale, boolean createSkeletons)
-            throws IOException {
-        write(srcDoc, doc, baseDir, bundleName, locale, true, createSkeletons);
-    }
-
-    private static void write(Resource srcDoc, final TranslationsResource doc,
-            final File baseDir, String bundleName, String locale, boolean utf8,
-            boolean createSkeletons) throws IOException {
+    //write translation
+    private static void write(final Resource srcDoc, final TranslationsResource doc,
```

Comment:
- [ ] why not just call the method `writeTranslation` so that no comment is needed?